navigation/navigation.css

```diff
@@ -0,0 +1,91 @@
+@import url(/style.css);
+@import url(/badge/badge.css);
+
+.nav-intro-heading,
+.nav-intro {
+  text-align: center;
+}
+.nav-container {
+  min-width: 40rem;
+  height: auto;
+  margin: 3rem 2rem 2rem 2rem;
+}
+.nav-standard-item {
```

Comment:
this property is just adding flex you may have a class - "flex-row" or "flex-column" in utility CSS and use that,

Reply:
yes, I will